src/api/gnosisProtocol/api.ts

```diff
@@ -251,3 +251,16 @@ function getBaseUrl(): string {
   // Production, staging, ens, ...
   return 'https://api.cow.fi/'
 }
+
+export function getPriceQuality(props: { fast?: boolean; verifyQuote: boolean }): /*PriceQuality*/ string {
```

Comment:
Because the enum type doesn't have `verified`, i need to update the SDK

Reply:
Check https://github.com/cowprotocol/cowswap/pull/2945/commits/6d61111fc0f0e7696e19b53418a3a7b0653c52ae now i can get rid of these things since th SDK is up to date